fineract-provider/src/main/java/org/apache/fineract/portfolio/loanaccount/loanschedule/service/LoanScheduleAssembler.java

```diff
@@ -336,7 +336,14 @@ private LoanApplicationTerms assembleLoanApplicationTermsFrom(final JsonElement
          */
         final DaysInMonthType daysInMonthType = loanProduct.fetchDaysInMonthType();
 
-        final DaysInYearType daysInYearType = loanProduct.fetchDaysInYearType();
+        DaysInYearType daysInYearType = null;
```

Comment:
Nit pick: remove the = null, it's not needed (because it gets assigned in either case), nicer to read, AND safer (if the code below EVER changes, it would cause a compilation failure instead of potential NPE). Makes sense?

Reply:
@davidyaha it looks like you haven't had time to address this feedback. As it's only minor, I'll not let this hold up merging.